fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/character/Resurrection.java

```diff
@@ -0,0 +1,6 @@
+package ch.bbw.fabbwled.lands.character;
+
+import ch.bbw.fabbwled.lands.book.SectionId;
+
+public record Resurrection(SectionId sectionIdToTurnTo, boolean isResurrectionPossible) {
```

Comment:
where is `isResurrectionPossible` needed? I though resurrections anyway go to a specific section.
we additionally need a way to trigger a resurrection via FE: so this likely needs a new REST endpoint. In a UI this could mean going to the inventory an clicking on the resurrection deal... maybe...